### ISSUE TITLE
Empty lines are not processed correctly

### ISSUE THREAD

**Issue by fluviusmagnus** [Vinzent03/obsidian-sort-and-permute-lines#5]
This happens if strict markdown style is applied to create new lines. E.g.:

Sort the lines below in alphabetic order

<pre>c

b

a</pre>

and we get

<pre>
 
 
a
b
c</pre>

Maybe an option could be provided to determine the behavior of the extension.

**Comment by ccchan234** [comment]
agree, this plugin destroyed any INFORMATION that is build using empty lines.

any way to bypass this?

thanks
